core/templates/services/utils.service.ts

```diff
@@ -24,7 +24,7 @@ import { downgradeInjectable } from '@angular/upgrade/static';
   providedIn: 'root'
 })
 export class UtilsService {
-  isEmpty(obj: any): boolean {
+  isEmpty(obj: Object): boolean {
```

Comment:
Ditto here.

Reply:
Done!

Reply:
I do not see any change/explanation here.  What is done?

Reply:
Ditto.

Reply:
Done

Reply:
Add some example here to clarify what is anything.